src/helpers/index.ts

```diff
@@ -0,0 +1,35 @@
+// export function getContractType(serviceType: string | undefined) {
+//   switch (serviceType) {
+//     case "CONX":
+//       return "coin";
+
+//     case "DRIVE":
+//       return "drive";
+
+//     case "ENGINE":
+//       return "engine";
+
+//     default:
+//       return "basic";
+//   }
+// }
```

Comment:
The commented code here is for the production version, which only has the three contract types.
The uncommented code is for the test server which has a million contract types.